packages/frameworks/logos/astro.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 192 256"><defs/><path fill="url(#paint0_linear)" fill-rule="evenodd" d="M131 19c2 2.3 3 5.6 5 12.1l43.2 142.3a180.3 180.3 0 00-51.7-17.6L99.3 60.6a3.7 3.7 0 00-7 0l-28 95.2a180.2 180.2 0 00-52 17.6L56 31c2-6.5 3-9.8 4.9-12.2a16 16 0 016.5-4.8c2.9-1.1 6.3-1.1 13-1.1h31.1c6.8 0 10.2 0 13.1 1.1A16 16 0 01131 19z" clip-rule="evenodd"/><path fill="#FF5D01" fill-rule="evenodd" d="M136.2 180.2c-7.1 6-21.4 10.2-37.8 10.2-20.2 0-37-6.3-41.5-14.7-1.6 4.8-2 10.4-2 14 0 0-1 17.3 11 29.3 0-6.2 5.1-11.3 11.4-11.3 10.7 0 10.7 9.4 10.7 17v.6c0 11.6 7 21.5 17 25.7a23.3 23.3 0 01-2.3-10.2c0-11 6.5-15.2 14-20 6-3.7 12.6-8 17.2-16.4a31 31 0 003.8-14.8c0-3.3-.6-6.5-1.5-9.4z" clip-rule="evenodd"/><path fill="url(#paint1_linear)" fill-rule="evenodd" d="M136.2 180.2c-7.1 6-21.4 10.2-37.8 10.2-20.2 0-37-6.3-41.5-14.7-1.6 4.8-2 10.4-2 14 0 0-1 17.3 11 29.3 0-6.2 5.1-11.3 11.4-11.3 10.7 0 10.7 9.4 10.7 17v.6c0 11.6 7 21.5 17 25.7a23.3 23.3 0 01-2.3-10.2c0-11 6.5-15.2 14-20 6-3.7 12.6-8 17.2-16.4a31 31 0 003.8-14.8c0-3.3-.6-6.5-1.5-9.4z" clip-rule="evenodd"/><defs><linearGradient id="paint0_linear" x1="144.6" x2="95.8" y1="5.4" y2="173.4" gradientUnits="userSpaceOnUse"><stop stop-color="#000014"/><stop offset="1" stop-color="#150426"/></linearGradient><linearGradient id="paint1_linear" x1="168.3" x2="126.1" y1="130.5" y2="219" gradientUnits="userSpaceOnUse"><stop stop-color="#FF1639"/><stop offset="1" stop-color="#FF1639" stop-opacity="0"/></linearGradient></defs></svg>
```

Comment:
I think this image needs to be a square, otherwise it will get cropped when we turn it into a circle.

For example 
![image](https://user-images.githubusercontent.com/229881/127046427-4255e5d2-ce75-4e3f-9e3e-bddebc1c83d7.png)

Reply:
I have made it square and re-centered it. Please let me know if the way I did so will be a problem.